### PR TITLE
feat: added Images and Input

### PR DESCRIPTION
**Pull request by Creator-singh**

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -2,8 +2,8 @@
     padding: 3rem;
```
Comment (aniketsingh13) [inline components/alert/alert.css:2]: you can make utility class for this

File: components/button/button.css [hunk 2]
```diff
@@ -2,8 +2,8 @@
     padding: 3rem;
 }
 
-#button p{
-    padding: 1rem 0;
+#button :where(h2,p){
+    padding: 1rem;
 }
```
Comment (aniketsingh13) [inline components/button/button.css:8]: remove space

File: components/images/images.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
+#image{
+    padding: 3rem;
+}
+
+#image :where(h2,p){
+    padding: 1rem;
+}
+
+.image-example{
+    display: flex;
```
Comment (aniketsingh13) [inline components/images/images.css:10]: you can also make utility class for this

File: components/images/images.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    justify-content: center;
+    border: 1px solid var(--lightgrey);
+    padding: 1.5rem;
+    box-shadow: 3px 3px 3px 3px var(--lightgrey);
+}
+
+.img-component{
+    height: 30rem; 
+    width: 30rem;
+    margin: 2rem;
```
Comment (aniketsingh13) [inline components/images/images.css:21]: you can also put it in the utility

**Review by aniketsingh13 — APPROVED** [review]
consider the changes  other thing good to go

**Review by laxmikanthkonthum — APPROVED** [review]
Nicely done

**Pull request merged**